purescript-cabal-parser/spago.dhall

```diff
@@ -11,7 +11,17 @@ When creating a new Spago project, you can use
 to generate this file without the comments in this block.
 -}
 { name = "cabal-parser"
-, dependencies = [ "console", "effect", "prelude", "psci-support" ]
-, packages = ./packages.dhall
-, sources = [ "src/**/*.purs", "test/**/*.purs" ]
+
+-- , dependencies = [ "console", "effect", "prelude", "psci-support" ]
+, dependencies = [] : List Text
+
+-- , packages = ./packages.dhall
+, packages = {=}
```

Comment:
This means that `dependencies` and `packages` are both empty.  

`packages` defines our package set (similar to a stackage resolver).  We don't have one, so it is empty.

`dependencies` defines the dependencies we use from `packages`.  This is also empty, since we need to implement everything from scratch.